cmake_ext.py

```diff
@@ -101,20 +101,10 @@ def build_extension(self, ext):
             f"-DCMAKE_BUILD_TYPE={cfg}",
         ]
 
-        # Read cmake args from extra.cfg
-        extra_cfg = cwd / "extra.cfg"
-        if extra_cfg.exists():
-            with open(extra_cfg) as f:
-                for model in f:
-                    model = model.strip()
-                    if not model:
-                        continue
-                    if "=" in model:
-                        if model[-1] == "/":
-                            model = model[:-1]
-                        cmake_args.append(f"-DBUILD_{model}")
-                    else:
-                        cmake_args.append(f"-DBUILD_{model}=ON")
+        models = get_models()
+        for model, path in models.items():
+            if path is not None:
+                cmake_args.append(f"-DBUILD_{model}={Path(path)}")
 
```

Comment:
With this code it is not possible to build sib23c00, sib23c02, sib23c03 by 2 reasons:
* "-DBUILD_sib23c00", ... are not defined
* option(BUILD_sib23c00 OFF) is removed from CMakeLists.txt

Reply:
Yes, that was intentional, but I forgot to remove the corresponding ifs.

Reply:
We only need the BUILD_X flags for models X which use external source files that are not part of the repo.